<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedusaExporter</class>
 <widget class="QWidget" name="MedusaExporter">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medusa Exporter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(91,91,91);
color: white*/
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="dataBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Data Export</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Destination folder:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="QPushButton" name="extractButton">
             <property name="text">
              <string>Export data</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="extractToLineEdit"/>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="extractToButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QGroupBox" name="compressOptionBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Compress options</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QRadioButton" name="separateRadio">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Separate files</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="togetherRadio">
                <property name="text">
                 <string>One file</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QGroupBox" name="compressBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Compress</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="imagesCheck">
                <property name="text">
                 <string>Images</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="metaCheck">
                <property name="text">
                 <string>Metadata</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="copyLocalBox">
             <property name="text">
              <string>Copy local annotations</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="atributesBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Atributes Export</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="exportFromLineEdit">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="3">
            <widget class="QPushButton" name="exportButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Export attributes</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QComboBox" name="curveDescriptionComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Pixel by pixel</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Key points</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Curve description</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QToolButton" name="exportFromButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QComboBox" name="chooseUserComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Curve density</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="skipIdenticalComboBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Skip identical</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Export from:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Choose user</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QComboBox" name="chooseFormatComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Output format</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QSpinBox" name="curveDensitySpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="normalizePointsCheck">
             <property name="text">
              <string>Normalize annotation points</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QGroupBox" name="recreateGroup">
             <property name="title">
              <string>Recreate annotation images</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="3" column="0" colspan="2">
               <widget class="QCheckBox" name="mergeButton">
                <property name="text">
                 <string>Merge with original image</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="bonesButton">
                <property name="text">
                 <string>Draw bones</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="skinButton">
                <property name="text">
                 <string>Draw skins</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="inflammationButton">
                <property name="text">
                 <string>Draw inflammations</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="jointsButton">
                <property name="text">
                 <string>Draw joints</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="miscBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Miscellaneous</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="openMedusaFolderButton">
             <property name="text">
              <string>Open Medusa export folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearMedusaFolderButton">
             <property name="text">
              <string>Clear Medusa export folder</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="progressBarLabel">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../dicom/ui/dicomRC.qrc"/>
  <include location="medusaExporterRC.qrc"/>
  <include location="C:/Users/Wojciech/.designer/backup/medusaExporterRC.qrc"/>
 </resources>
 <connections/>
</ui>
